astro/src/components/listing/entry.astro

```diff
@@ -1,16 +1,51 @@
 ---
+import moment from 'moment';
+import 'moment/locale/it';
+import i18next, { t } from "i18next";
+
+const today = moment();
+moment.locale('en'); 
+
 export interface Props {
 	report: Object;
 }
 
 const report = Astro.props.report
+
+const givenDate = moment(report.theft_date);
+const currentDate = moment();
+const duration = moment.duration(currentDate.diff(givenDate));
+const daysAgo = duration.asDays();
+
+const formattedRelativeDate = daysAgo < 0
+  ? moment.duration(-daysAgo, 'days').humanize(true) + ' from now'
+  : moment.duration(daysAgo, 'days').humanize(true) + ' ago';
+
+const formattedDate = moment(report.theft_date, 'D-M-YYYY').format('MMMM Do, YYYY');
```

Comment:
these strings here, they depend very much on the selected language, we cannot hardcode like this